compiler/src/dotty/tools/dotc/transform/Pickler.scala

```diff
@@ -1,4 +1,5 @@
-package dotty.tools.dotc
+package dotty.tools
+package dotc
```

Comment:
Spurious changes only in this file.

Reply:
No, just making the change so next time I need access to the utilities in the tools package (like I did during the development of this feature), I don't need to re-change the packages.